### PR TITLE
plugin-torture: git 2013-10-03 -> 5

### PR DESCRIPTION
**Pull request by magnetophon**
###### Motivation for this change

update pkg
###### Things done
- [x] Tested using sandboxing
  ([nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS,
    or option `build-use-chroot` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS 16.03
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/audio/plugin-torture/default.nix [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
   };
 
   buildInputs = [ boost ladspaH lilv lv2 pkgconfig serd sord sratom ];
 
   installPhase = ''
     mkdir -p $out/bin
     cp plugin-torture $out/bin/
-    cp README $out/bin/
+    cp find-safe-plugins $out/bin/
+    cp README.md $out/bin/
```
Comment (matthewbauer) [inline pkgs/applications/audio/plugin-torture/default.nix]: Does this need to be in $out/bin/?

Comment (magnetophon) [inline pkgs/applications/audio/plugin-torture/default.nix]: @matthewbauer no...

File: pkgs/applications/audio/plugin-torture/default.nix [hunk 2]
```diff
@@ -1,21 +1,22 @@
-{ stdenv, fetchgit, boost, ladspaH, lilv, lv2, pkgconfig, serd, sord, sratom }:
+{ stdenv, fetchFromGitHub, boost, ladspaH, lilv, lv2, pkgconfig, serd, sord, sratom }:
 
 stdenv.mkDerivation rec {
-  name = "plugin-torture-git-${version}";
-  version = "2013-10-03";
+  name = "plugin-torture-v${version}";
```
Comment (rasendubi) [inline pkgs/applications/audio/plugin-torture/default.nix]: `v` is not needed.

Comment (magnetophon) [inline pkgs/applications/audio/plugin-torture/default.nix]: @rasendubi The upstream release name has a v.  Removed it anyway.

@matthewbauer , @rasendubi : Let me know when we're ready for squashing!

Comment (rasendubi) [inline pkgs/applications/audio/plugin-torture/default.nix]: You'd better squash after every change (or rather just commit amend). As your PR is small, that doesn't impede review.

**Comment by magnetophon** [comment]
The pkg has linux as a platform, how come [travis seems to fail because of an OSX error](https://travis-ci.org/NixOS/nixpkgs/builds/145191148)?
Or am I misinterpreting that page?

**Comment by rasendubi** [comment]
`platform` is mostly for Hydra. I don't think Travis respects this field.

**Comment by magnetophon** [comment]
@matthewbauer @rasendubi  @joachifm Is this one basically just waiting for a squash?

**Comment by joachifm** [comment]
Looks superficially okay to me. Please squash.

**Comment by magnetophon** [comment]
@joachifm OK, thanks.  Done.

**Pull request merged**